pom.xml

```diff
@@ -938,6 +940,99 @@
         <mr.bundle.avro.shade.prefix>org.apache.hudi.</mr.bundle.avro.shade.prefix>
       </properties>
     </profile>
+    <profile>
+      <id>unijavadoc</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-compiler-plugin</artifactId>
+            <configuration>
+              <source>1.8</source>
```

Comment:
hard code `1.8`

Reply:
Good catch.  Made this config in one place with a property.